common/pom.xml

```diff
@@ -105,6 +105,29 @@
             </configuration>
           </execution>
         </executions>
+        <!-- Needed to support JDK9, https://issues.apache.org/jira/browse/MSHADE-242 -->
```

Comment:
create a followup issue to cleanup and update dependencies when this external issue is resolved?